ledger/catchpointtracker_test.go

```diff
@@ -1055,6 +1170,15 @@ func TestSecondStagePersistence(t *testing.T) {
 		t, ml, cfg, filepath.Join(tempDirectory, config.LedgerFilenamePrefix))
 	defer ct.close()
 
+	isCatchpointRound := func(rnd basics.Round) bool {
```

Comment:
there is some code duplication but I prefer to have `isCatchpointRound` with a single arg and not 3 args